components/script/dom/webglrenderbuffer.rs

```diff
@@ -33,6 +35,7 @@ impl WebGLRenderbuffer {
             ever_bound: Cell::new(false),
             is_deleted: Cell::new(false),
             renderer: renderer,
+            internalformat: Cell::new(None),
```

Comment:
I think the usual way to name members is using snake case (i.e. `internal_format`). Not a huge deal though.